test/parallel/test-https-simple.js

```diff
@@ -36,14 +36,8 @@ const options = {
   cert: fs.readFileSync(common.fixturesDir + '/keys/agent1-cert.pem')
 };
 
-const tests = 2;
-let successful = 0;
-
 const testSucceeded = function() {
-  successful = successful + 1;
-  if (successful === tests) {
-    server.close();
-  }
+  server.close();
```

Comment:
When this is run later, it picks up the `server` variable still? 🤔😖 

Maybe we should make this into a function declaration (no assignment) and put it at the bottom of the file for clarity.